ChemistryLib/PhreeqcIO.h

```diff
@@ -101,8 +101,6 @@ class PhreeqcIO final : public ChemicalSolverInterface
     std::unique_ptr<Output> const _output;
     std::unique_ptr<Dump> const _dump;
     Knobs const _knobs;
-    std::vector<std::pair<int, std::string>> const&
-        _process_id_to_component_name_map;
     double _dt = std::numeric_limits<double>::quiet_NaN();
     const int phreeqc_instance_id = 0;
```

Comment:
Although not from this PR,  the underscore is not used as the prefix for this member name.  I will change it in my PR.

Reply:
@wenqing  There are many more such cases. Don't mix up different topics in single PR or commit.

Reply:
Not yet. Just want to do after the PR for name changing is merged or its current review is done.